### PR TITLE
schedule: add more information for split operator

### PR DESCRIPTION
**Pull request by bufferflies**
Signed-off-by: bufferflies <1045931706@qq.com>

### What problem does this PR solve?

Issue Number:ref #4877

### What is changed and how does it work?

```commit-message
```

### Check List

Tests

- Integration test

Code changes

Side effects

Related changes

### Release note

```release-note
 None.
```

### CODE REVIEW

File: server/schedule/operator/create_operator.go [hunk 1]
```diff
@@ -129,11 +128,14 @@ func CreateSplitRegionOperator(desc string, region *core.RegionInfo, kind OpKind
...
 		for i := range keys {
-			hexKeys[i] = hex.EncodeToString(keys[i])
+			hexKeys[i] = core.HexRegionKeyStr(keys[i])
 		}
 		brief += fmt.Sprintf(" and keys %v", hexKeys)
 	}
-	return NewOperator(desc, brief, region.GetID(), region.GetRegionEpoch(), kind|OpSplit, region.GetApproximateSize(), step), nil
+	op := NewOperator(desc, brief, region.GetID(), region.GetRegionEpoch(), kind|OpSplit, region.GetApproximateSize(), step)
+	op.AdditionalInfos["region-start-key"] = core.HexRegionKeyStr(region.GetStartKey())
+	op.AdditionalInfos["region-start-key"] = core.HexRegionKeyStr(region.GetEndKey())
```
Comment (lhy1024) [inline server/schedule/operator/create_operator.go:137]: end-key

**Review by AndreMouche — APPROVED** [review]
LGTM

**Review by lhy1024 — APPROVED** [review]

File: server/schedule/operator/create_operator.go [hunk 2]
```diff
@@ -129,11 +128,14 @@ func CreateSplitRegionOperator(desc string, region *core.RegionInfo, kind OpKind
...
 		hexKeys := make([]string, len(keys))
 		for i := range keys {
-			hexKeys[i] = hex.EncodeToString(keys[i])
+			hexKeys[i] = core.HexRegionKeyStr(keys[i])
 		}
 		brief += fmt.Sprintf(" and keys %v", hexKeys)
 	}
-	return NewOperator(desc, brief, region.GetID(), region.GetRegionEpoch(), kind|OpSplit, region.GetApproximateSize(), step), nil
+	op := NewOperator(desc, brief, region.GetID(), region.GetRegionEpoch(), kind|OpSplit, region.GetApproximateSize(), step)
+	op.AdditionalInfos["region-start-key"] = core.HexRegionKeyStr(region.GetStartKey())
```
Comment (rleungx) [inline server/schedule/operator/create_operator.go:136]: Does it need redaction?

Comment (bufferflies) [reply server/schedule/operator/create_operator.go:136]: yes

File: server/schedule/operator/create_operator.go [hunk 3]
```diff
@@ -128,13 +129,13 @@ func CreateSplitRegionOperator(desc string, region *core.RegionInfo, kind OpKind
...
 		for i := range keys {
-			hexKeys[i] = core.HexRegionKeyStr(keys[i])
+			hexKeys[i] = core.HexRegionKeyStr(logutil.RedactBytes(keys[i]))
 		}
 		brief += fmt.Sprintf(" and keys %v", hexKeys)
 	}
 	op := NewOperator(desc, brief, region.GetID(), region.GetRegionEpoch(), kind|OpSplit, region.GetApproximateSize(), step)
-	op.AdditionalInfos["region-start-key"] = core.HexRegionKeyStr(region.GetStartKey())
-	op.AdditionalInfos["region-end-key"] = core.HexRegionKeyStr(region.GetEndKey())
+	op.AdditionalInfos["region-start-key"] = core.HexRegionKeyStr(logutil.RedactBytes(region.GetStartKey()))
```
Comment (rleungx) [inline server/schedule/operator/create_operator.go:137]: If the redaction is enabled, is there necessary to use `HexRegionKeyStr`?

**Pull request merged**